File: Data Structures & Algorithms/reverse-integer/submission-0.py
class Solution:
    def reverse(self, x: int) -> int:
        num = (str(x)[::-1])
        if num[len(num)-1] == "-":
            num = - int(num[0:len(num)-1])
        if int(num) > 2**31 - 1 or int(num) < -2**31:
            return 0
        return int(num) 

File: Data Structures & Algorithms/largest-3-same-digit-number-in-string/submission-0.cpp
class Solution {
public:
    string largestGoodInteger(string num) {
        string ans = "";
        bool f = false;
        char n = '0';
        for (int i = 1; i < num.length() - 1; i++) {
            if ((num[i] == num[i-1] && num[i+1] == num[i]) && num[i] >= n) {
                n = num[i];
                f = true;
            }
        }
        if (f) {
            for (int i = 0; i < 3; i++) {
                ans += n;
            }
        }
        return ans;
    }
};